spec/models/user_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe User, type: :model do
+  context 'validations' do
+    it 'name should be present' do
+      user = User.new(name: nil)
+      expect(user.valid?).to eq(false)
```

Comment:
- [ ] **[OPTIONAL]** You could write this spec in a better readable way using the `be_valid` helper:

```suggestion
      expect(user).not_to be_valid
```